clippy_lints/src/ptr.rs

```diff
@@ -44,13 +44,30 @@ declare_lint! {
     "comparing a pointer to a null pointer, suggesting to use `.is_null()` instead."
 }
 
+/// **What it does:** This lint checks for functions that take immutable refs and return
```

Comment:
s/ref/reference
I don't like abbreviations in documentation and I've seen people being confused between `&` and the keyword `ref` to many times.